manifests/dispatch_router.pp

```diff
@@ -30,8 +34,13 @@
   String $logging_level = 'info+',
   Enum['file', 'syslog'] $logging = 'syslog',
   Stdlib::Absolutepath $logging_path = '/var/log/qdrouterd',
+
+  Enum['present', 'absent'] $ensure = 'present',
 ) {
 
+  class { 'qpid::router':
+    ensure => $ensure,
+  }
   contain qpid::router
 
   include certs::qpid_router::server
```

Comment:
Perhaps this should be inside `if $ensure == 'present' {}`. However, that also implies the `ssl_profile` should be hidden. That also makes me wonder if all router statements (listener and log) should be hidden.